roles/osp_observability/tasks/configure_ceilometer.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: get service facts
+  service_facts:
+
+- name: check if ceilometer service is deployed
+  fail:
+    msg: Invalid Env, it doesn't contain ceilometer.
+  when: ansible_facts.services["tripleo_ceilometer_agent_notification.service"] is not defined
+
+- name: Create ceilometer pipeline.yml file
+  template:
+    src: ceilometer_pipeline.yml.j2
+    dest: "{{ ceilometer_pipeline_path }}"
```

Comment:
Is it okay for us to override the entire contents of this file? Could it be that users have configured ceilometer pipelines via tripleo that we should not be clobbering?

Reply:
Agree, we should just switch the sinks to Jaromir's TCP publisher. Will work on that.